data/data/agent/systemd/units/agent-add-node.service

```diff
@@ -1,10 +1,10 @@
 [Unit]
 Description=Adds the current node to an already existing cluster
 Wants=network-online.target
-Requires=apply-host-config.service
+Requires=agent-auth-token-status.service apply-host-config.service
```

Comment:
Requires is a very strong dependency, I suspect Wants would be sufficient here.

There's also something to be said for using `WantedBy`/`Before` in `agent-auth-token-status.service` rather than listing it in every unit that uses a token.

Reply:
@zaneb I added it here https://github.com/openshift/installer/pull/8802/commits/4c01a1ce58dfae7266c768c4622421a7e4c3e7f4